.github/workflows/CI.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           override: true
 
       - name: "${{ matrix.conf.name }}: tests"
-        run: xvfb-run --auto-servernum cargo test --manifest-path ${{ matrix.conf.name }}/Cargo.toml --features "${{ matrix.conf.features }}"
+        run: xvfb-run --auto-servernum cargo test --manifest-path ${{ matrix.conf.name }}/Cargo.toml --features "${{ matrix.conf.features }}" -- --test-threads 1
```

Comment:
Could we use `serial_test` like in `gio` for this instead?

Reply:
That sadly doesn't work, despite tests being run one after another, they are still run in different threads
```
thread 'expression::tests::test_expressions' panicked at 'Attempted to initialize GTK from two different threads.', gtk4/src/rt.rs:98:9
```
and I don't think there's a way to "uninitialize"  gtk at the end of each test so that the test run after can work.

Reply:
You could maybe run a single worker thread that initializes GTK and then from each test only queue up a job there to run the test.

Reply:
That worked fine :) thanks!